corehq/apps/hqwebapp/templates/hqwebapp/base.html

```diff
@@ -413,6 +396,12 @@
           callback: function (ko, mapping) {
             ko.mapping = mapping;
           },
+          paths: {"modernizr": "{% static 'hqwebapp/js/lib/modernizr' %}"},
```

Comment:
What is this for? The `build_requirejs` step in the deploy process uses the `paths` config to point each js file to the correct version on our CDN (docs [here](https://github.com/dimagi/commcare-hq/blob/master/docs/js-guide/static-files.md#cache-busting)), so this might get overwritten.

Reply:
@mjriley I think adding modernizr to [this bit of code](https://github.com/dimagi/commcare-hq/blob/08640a777f11c28813cf242b7417b1e850b59c4c/corehq/apps/hqwebapp/static/hqwebapp/js/hqModules.js#L58-L67) (as jenny mentioned above) will make this line unnecessary :)

Reply:
Possibly irrelevant with new updates to this PR, but just realized I think the issue here is that the dependency in `main.js` should be `hqwebapp/js/lib/modernizr` not just `mordernizr`.

Reply:
No, you were correct -- thanks for pointing that out, as I was wrong about the script tag bit. The path was just how to locate the file if it wasn't stored in the root directory, and you made the correct observation that I just needed to give an absolute path here.